docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py makemigrations &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+
+  db:
+    image: postgres:16.0-alpine3.17
+    restart: always
+    env_file:
+      - .env

```

Comment:
Verify that all necessary environment variables are defined in the `.env` file to ensure proper configuration of both the Django application and the PostgreSQL database.